src/main/webapp/scripts/region-selector.js

```diff
@@ -0,0 +1,17 @@
+function getRegions() {
+  return gapi.client.youtube.i18nRegions.list({
+    'part': [
+      'snippet'
+    ],
+    'hl': 'es_MX'
+  })
+  .then((response) => {
+    return response.result;
+  })
+  .catch((error) => {
+    console.error('An error occurred getting the region list:', error);
+    throw error;
```

Comment:
doing a console.error and throwing the error will make it have two logs in the console. If the error is informative enough then you don't need to catch. For future PR, maybe you want to catch it to have a different behavior on the UI? Like not letting them submit anything.